common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -103,30 +104,62 @@ public static String modifyMultipleMatchKeyword(String query){
 
                     // Optionally filter on standard concept, vocabulary ID, domain ID
                     if (standardConceptFilter.equals(StandardConceptFilter.STANDARD_CONCEPTS)) {
+
+                        predicates.add(
```

Comment:
It might be simpler to make a conceptNamePredicate used here and in the next if block; and only make conceptName_filter in the else case.